### ISSUE TITLE
Should we increase level of detail in the image on landing pages?

### ISSUE THREAD

**Issue by tdbeattie** [openpracticelibrary/openpracticelibrary#303]
**Problem:**

People do not see the full canvas that the Open Practice Library is visualised on when they first land on the page and when they read the "About" section.

Possibly using one of the more detailed images which includes, for example, the Discovery loop split in the Why/Who and Outcomes area and the Foundation split into the Culture/Collaboration and Technical area, will have a greater impact and understanding behind this.

Keen to get others people's thoughts on this.

**Comment by rdebeasi** [comment]
I'm assuming that this is referring to the main "mobius loop" on the homepage, right?

We redesigned that image to be small and concise based on feedback that the image was too large and pushed the content down (#263, and also usability tests). I definitely agree that we should make it easier to understand this canvas, but I want to make sure that we don't accidentally make the homepage confusing instead.

We ran into something similar in the Boston enablement last week, we got a lot of questions about how the "big picture" of the Mobius loop looks in real life. It wasn't so much that participants couldn't understand which practices went where; it was more that they weren't "bought in" to the concept as a whole.

It would be tempting to try and fix this by shrinking that Mobius poster on the wall of the London office down and stick it in the page, but I think that graphic works better as a giant poster than as a screen-sized graphic. (Imagine it on a phone! :stuck_out_tongue_closed_eyes: )

Anyway, I know that this is a big giant "yes and", but here's the big question that I have regarding the homepage: How might we add meaning while also reducing noise?
